### PR TITLE
Added bike.js

### PR DESCRIPTION
**Pull request by kanompia**
Adding bike function.

### CODE REVIEW

**Comment by kanompiagmail** [comment]
Ok, go ahead.

File: bike.js [hunk 1]
```diff
@@ -0,0 +1 @@
+const my_vars = "bike";
```
Comment (kanompiagmail) [inline bike.js:1]: this need to say BIKE.

Comment (kanompiagmail) [inline bike.js:1]: bla bla bla

**Review by kanompiagmail — CHANGES_REQUESTED** [review]
Please fix this.

File: bike.js (hunk 1 above)
Comment (kanompia) [reply bike.js:1]: problem fixed

**Review by kanompiagmail — APPROVED** [review]
Approved.

**Pull request merged**